src/v2/cookbook/debugging-in-vscode.md

```diff
@@ -72,65 +90,71 @@ Click on the Debugging icon in the Activity Bar to bring up the Debug view, then
+Hay otros modelos de depuración, variando en complejidad. El más simple y popular
+es usar el excelente Vue.js devtools [para Chrome](https://chrome.google.com/webstore/detail/vuejs-devtools/nhdogjmejiglipccpnnnanhbledajbpd) y [para Firefox](https://addons.mozilla.org/en-US/firefox/addon/vue-js-devtools/).
+Algunos de los beneficios de trabajar con las devtools es que le permite editar
+las propiedades y ver los cambios reflejados en tiempo real. El otro gran
+beneficio es la habilidad de depurar viajando en el tiempo para Vuex.
 
-![Devtools Timetravel Debugger](/images/devtools-timetravel.gif)
+![Devtools Depurador viajando en el tiempo](/images/devtools-timetravel.gif)
 
-<p class="tip">Please note that if the page uses a production/minified build of Vue.js (such as the standard link from a CDN), devtools inspection is disabled by default so the Vue pane won't show up. If you switch to an unminified version, you may have to give the page a hard refresh to see them.</p>
+<p class="tip">
+  Por favor tenga presente que si la página usa una compilación minificada de
+  producción de Vue.js (como ser el link estándar del CDN), la inspección de
+  devtools estará deshabilitada por defecto lo cual no mostrará el panel de Vue.
+  Si se cambia a una versión no minificada, quizás sea necesario que recargue la
+  página forzadamente para poder verlo.
+</p>
 
-### Vuetron
+### Simple Sentencia Debugger
 
-[Vuetron](http://vuetron.io/) is a really nice project that extends some of the work that vue-devtools has done. In addition to the normal devtools workflow, you are able to:
-
-* Quickly view API Request/Response: if you're using the fetch API for requests, this event is displayed for any request sent. The expanded card displays the request data as well as the response data.
-* Subscribe to specific parts of your application’s state for faster debugging
-* Visualize component hierarchy, and an animation allows you to collapse or expand the tree for specific hierarchy views.
-
-![Vuetron Hierarchy](/images/vuetron-hierarchy.gif)
-
-### Simple Debugger Statement
-
-The example above has a great workflow. However, there is an alternative option where you can use the [native debugger statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/debugger) directly in your code. If you choose to work this way, it's important that you remember to remove the statements when you're done.
+El ejemplo anterior tiene un gran flujo de trabajo. Sin embargo, hay una
+opción alternativa donde puede usar la [sentencia nativa debugger](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/debugger)
+directamente en su código. Si elige trabajar de esta forma, es importante que
+recuerde remover las sentencias una vez haya terminado.
 
 ```js
 <script>
 export default {
```

Comment:
@FLarra  sorry, I didn't tell you this, it was my mistake. The code must be kept in English, we can translate messages and comments. In this case it could be as follows.

export default {
  data() {
    return {
      message: ''
    }
  },
  mounted() {
    const hello = 'Hola Mundo!'
    debugger
    this.message = hello
  }
};